groups/cdn/cloudfront.tf

```diff
@@ -53,6 +53,31 @@ resource "aws_cloudfront_origin_access_control" "assets" {
   signing_behavior                  = "always"
   signing_protocol                  = "sigv4"
 }
+resource "aws_s3_bucket" "log_bucket" {
+  bucket = "cloudfront-logs-bucket"
+
+  block_public_acls   = true
+  block_public_policy = true
+
+  lifecycle_rule {
+    id      = "log-retention"
+    enabled = true
+
+    expiration {
+      days = 30
+    }
+  }
+
+  tags = {
+    Name = "Cloudfront Logs Bucket"
+  }
+}
```

Comment:
Agree that the input args do not appear to be valid. Please also note that we would required adequate encryption of the bucket via implementing a server_side_encryption_configuration

We would also require that all requests are restricted to SSL via implementing AllowSSLRequestsOnly via a bucket policy

including enforcing the following:

  block_public_acls       = true
  block_public_policy     = true
  ignore_public_acls      = true
  restrict_public_buckets = true